spec/requests/api/v1/articles/show_spec.rb

```diff
@@ -40,4 +45,13 @@
       expect(response_json["error_message"]).to eq "Sorry we can not find that article"
     end
   end
+
+  describe "Visitor can only see part of premium content" do
```

Comment:
Please test that they can still see the full content of a none premium article

Reply:
Will do!